app/src/main/java/com/easyfitness/utils/AlarmReceiver.java

```diff
@@ -4,24 +4,36 @@
 import android.content.Context;
 import android.content.Intent;
 import android.media.AudioAttributes;
+import android.media.AudioManager;
+import android.media.ToneGenerator;
 import android.os.Build;
 import android.os.VibrationEffect;
 import android.os.Vibrator;
+import android.util.Log;
 
 public class AlarmReceiver extends BroadcastReceiver {
     @Override
-    public void onReceive(Context context, Intent arg1) {
+    public void onReceive(Context context, Intent intent) {
         long[] pattern = {
                 0,  // Start immediately
                 500, 500, 500, 500, 500};
         Vibrator v = (Vibrator) context.getSystemService(Context.VIBRATOR_SERVICE);
+        ToneGenerator toneGen = new ToneGenerator(AudioManager.STREAM_ALARM, 100);
+
+        // 3 beeps of each 500 ms
+        if (intent.getBooleanExtra("playSoundAfterRestTimer", true)) {
+            toneGen.startTone(ToneGenerator.TONE_SUP_BUSY, 3000);
```

Comment:
I find the sound a bit weird but I have nothing else to propose. So that's fine, it does the job! :)

Reply:
I totally agree with you. I tested multiple ones from the mentioned link and that one was actually the most appropriate one.
https://developer.android.com/reference/android/media/ToneGenerator#constants_1